handbook/product/README.md

```diff
@@ -16,13 +16,15 @@ The product team is responsible for product design tasks like drafting [changes
 
 At Fleet, like [GitLab](https://about.gitlab.com/handbook/product-development-flow/#but-wait-isnt-this-waterfall) and [other organizations](https://speakerdeck.com/mikermcneil/i-love-apis), every change to the product's UI gets [wireframed first](https://fleetdm.com/handbook/company/why-this-way#why-do-we-use-a-wireframe-first-approach).
 
-- Take the top issue that is assigned to you in the "Prioritized" column of the [drafting board](https://app.zenhub.com/workspaces/-product-backlog-coming-soon-6192dd66ea2562000faea25c/board).
+- Take the top user story that is assigned to you in the "Prioritized" column of the [drafting board](https://app.zenhub.com/workspaces/-product-backlog-coming-soon-6192dd66ea2562000faea25c/board).
   
 - Create a new file inside the [Fleet product](https://www.figma.com/files/project/17318630/%F0%9F%94%9C%F0%9F%93%A6-Fleet-EE%C2%AE-(product)?fuid=1234929285759903870) Figma project. See [Working with Figma](https://fleetdm.com/handbook/product#working-with-figma) below for more details.
   
--  Use dev notes (component available in our library) to highlight important information to engineers and other teammates.
+- Use dev notes (component available in our library) to highlight important information to engineers and other teammates.
 
-- Draft changes to the Fleet product that solve the problem specified in the issue. Constantly place yourself in the shoes of a user while drafting changes. Place these drafts in the appropriate Figma file in Fleet product project.
+- Draft changes to the Fleet product that solve the problem specified in the story. Constantly place yourself in the shoes of a user while drafting changes. Place these drafts in the appropriate Figma file in Fleet product project.
+
+- Be intentional about style changes that affect different parts of the product (e.g. button border-radius or modal width). These are made opportunistically as part of user stories. At Fleet we don't prioritize style-only user stories.
```

Comment:
```suggestion
- Implemented behavior that doesn't match the style guide or wireframes, or looks odd at certain viewport widths, is a bug, and should be filed ASAP, as precisely as possible (with limited scope) so the root cause can be addressed quickly, usually in the current sprint before there is time for further expensive bugs or misunderstandings to accumulate.
- UI changes are not bugs if the current behavior is what was wireframed and documented.
- Be intentional about style guide changes that affect more than one area of the product (e.g. button border-radius or modal width). These changes are important and valued, and should be written as user stories like any other.  But they are also expensive.  They require extra drafting to carefully identify every place which needs to be changed ahead of time, and then extra quality assurance to check that all those places were correctly updated.
```

Reply:
> These changes are important and valued, and should be written as user stories like any other.

Hey @mikermcneil this is not what @marko-lisica, @rachaelshaw, @mike-j-thomas, and I decided.

We decided to include style guide changes as part of stories. They don't get their own stories because these won't get prioritized in FF.

Let's discuss during the next design review. I'm closing this PR for now.

Reply:
>  because these won't get prioritized in FF.

This is what needs to change.

Reply:
>  because these won't get prioritized in FF.

This is what needs to change.  Like technical debt, UX debt will get prioritized if it drives value. 

An example of this is the recent work on "Queries", or the work we did at the top of 2023 to make the UI theme match the brand identity.

> Let's discuss during the next design review.
We can discuss at CEO office hours this evening, or sooner, making the decision and handbooking it ASAP.  This doesn't need to wait.  @zhumo @noahtalerman please join.

Reply:
Heads up, I won't be able to join CEO office hours this evening. Mo is planning on joining.

Reply:
Mo helped me see how I was somehow confused here.  Key part of my confusion was this sentence:
> At Fleet we don't prioritize style-only user stories.

Sorry for the confusion guys.

Reply:
All good! Thanks for getting us on a call to clear that up.